.cicd/eosio-contracts-build.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+set -eo pipefail
+. ./.cicd/helpers/general.sh
+CDT_DIR_HOST=$(pwd)
+mkdir -p build_eosio_contracts 
+
+
+[[ ! -z "$CONTRACTS_VERSION" ]] || export CONTRACTS_VERSION="$(cat "$PIPELINE_CONFIG" | jq -r '.dependencies["eosio.contracts"]')"
+git clone -b "$CONTRACTS_VERSION" https://github.com/EOSIO/eosio.contracts.git 
+
+if [[ $(uname) == 'Darwin' ]]; then
+    export PATH=$CDT_DIR_HOST/build/bin:$PATH
+    cd build_eosio_contracts
+    cmake $CDT_DIR_HOST/eosio.contracts
+    make -j$JOBS
+else #Linux
+    ARGS=${ARGS:-"--rm --init -v $(pwd):$MOUNTED_DIR"}
+    . $HELPERS_DIR/docker-hash.sh
+
+    PRE_CONTRACTS_COMMAND="export PATH=$MOUNTED_DIR/build/bin:$PATH && cd $MOUNTED_DIR/build_eosio_contracts"
+    BUILD_CONTRACTS_COMMAND="cmake $MOUNTED_DIR/eosio.contracts && make -j$JOBS"
+
+    # Docker Commands
+    if [[ $BUILDKITE == true ]]; then
```

Comment:
This guard can also be removed, as we are not invoking `buildkite-agent` here and it would make debugging easier if engineers can invoke these scripts locally.

Reply:
Done.